tests/unit_tests/columns/__init__.py

```diff
@@ -0,0 +1,16 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
is this file necessary?

Reply:
It is... in theory in Python 3 empty `__init__py` files are not needed, but in practice we had a lot of problems when we omit them.

For `pytest` in particular, if you have files with the same name in different directories inside your tests, like:

- `tests/unit_tests/datasets/test_model.py`
- `tests/unit_tests/tables/test_model.py`

Then you *need* to add the `__init__.py` file to the subdirectories, otherwise pytest will only read one of them.